### PR TITLE
feat: reduce navi and add community information

### PR DESCRIPTION
**Pull request by danielpeintner**
https://github.com/w3c/wot-marketing/issues/203

### CODE REVIEW

**Comment by egekorkan** [comment]
In the call of 02.11, there were the following comments:
- Standards sounds like standardization documents
- Resolution was to use the word `Groups` and also put the Community Group in there.

@danielpeintner can you incorporate these changes?

**Comment by danielpeintner** [comment]
> @danielpeintner can you incorporate these changes?

I updated the PR accordingly

grafik

**Review by egekorkan — APPROVED** [review]

File: docs/cg/index.html [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 group: "navigation"
 ---
 <div>
+	<h2>How to get in touch</h2>
+	<ul>
+		<li><a href="https://twitter.com/W3C_WoT">Twitter</a></li>
+		<li><a href="https://stackoverflow.com/questions/tagged/web-of-things">Stack Overflow</a></li>
+		<li>GitHub Discussions (<mark>TODO</mark>)</li>
+	</ul>
+
```
Comment (danielpeintner) [inline docs/cg/index.html:13]: Note: I don't think this PR is ready to be merged until we solved the aboce

File: docs/cg/index.html [hunk 2]
```diff
@@ -1,9 +1,16 @@
 ---
 layout: default
-title: W3C WoT – Community Groups
+title: W3C WoT – Community
```
Comment (danielpeintner) [inline docs/cg/index.html:3]: Shall we keep "Groups" in the title?

File: docs/_config.yml [hunk 3]
```diff
@@ -8,16 +8,14 @@ url: "https://www.w3.org"
...
-   url: ig/
- - title: Community Group
-   url: cg/
+ - title: Groups
+   subnavigation:
+   - title: Working Group
+     url: wg/
+   - title: Interest Group
+     url: ig/
+   - title: Community
```
Comment (danielpeintner) [inline docs/_config.yml:17]: Shall we keep "Groups" in the navigation?

**Comment by egekorkan** [comment]
My feedback:

- "How to get in touch" should be in another tab like "Contact" to the right of Videos
- "Community" should be named "Community Groups" to be aligned with the other items
- Contact information should include Public Emailing lists, IRC, and mentioning that GitHub issues can be used for contacting

**Comment by egekorkan** [comment]
From @ashimura : we should say that the CGs are related. They are not part of the WoT work officially.
From @relu91 : we should include the Web Thing Protocol Community Group

**Comment by danielpeintner** [comment]
> * "How to get in touch" should be in another tab like "Contact" to the right of Videos
> 
> * "Community" should be named "Community Groups" to be aligned with the other items

Done

> * Contact information should include Public Emailing lists, IRC, and mentioning that GitHub issues can be used for contacting

The contact page might need more work. Proposals what should be in there?


grafik

**Review by egekorkan — APPROVED** [review]

**Pull request merged**